content/docs/for-platform-operators/concepts/build-run-images.md

```diff
@@ -0,0 +1,94 @@
+
+<!--more-->
+
+Each buildpack checks the source code and provides any relevant dependencies. Then, a `base image` is then generated from the application’s source code and these dependencies.
+
+## Base image types
+
+A `base image` is an `OCI image` containing the base, or initial set of layers, for other images. It is helpful to distinguish between two distinct types of images, `Build` and `Run` images.
+
+### Build image
+
+A `build image` is an `OCI image` that serves as the base image for the `build` environment in which buildpacks `lifecycle` is executed.
+
+A typical `build image` might determine:
+
+* The OS distro in the build environment
+* OS packages installed in the build environment
+* Trusted CA certificates in the build environment
+* The default user in the build environment
+
+The platform must ensure that:
+
+* The image config's `User` field is set to a non-root user with a writable home directory
+* The image config's `Env` field has the environment variable `CNB_USER_ID` set to the user [UID/SID](https://github.com/buildpacks/spec/blob/main/README.md#operating-system-conventions) of the user specified in the `User` field
+* The image config's `Env` field has the environment variable `CNB_GROUP_ID` set to the primary group [GID/SID](https://github.com/buildpacks/spec/blob/main/README.md#operating-system-conventions) of the user specified in the `User` field
+* The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`)
+
+The platform should ensure that:
+
+* The image config's `Label` field has the label `io.buildpacks.base.maintainer` set to the name of the image maintainer.
+* The image config's `Label` field has the label `io.buildpacks.base.homepage` set to the homepage of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.released` set to the release date of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.description` set to the description of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.metadata` set to additional metadata related to the image.
+
+#### Anatomy of a build image
+
+Typically, a `build` image may include:
+
+* Shell
+* C-compiler
+* Minimal operating system distribution, such as Linux utilities that build systems might call out to
+* Build time libraries
+
+### Run image
+
+A `run image` is an `OCI image` that serves as the base image for the final application image.
+
+A typical run image might determine:
+
+* The OS distro or distroless OS in the launch environment
+* OS packages installed in the launch environment
+* Trusted CA certificates in the launch environment
+* The default user in the run environment
+
+The platform must ensure that:
+
+* The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`)
```

Comment:
Again, I wonder if "`Env` field" is the correct term here.  Do we mean that the run image must define a `PATH` environment variable?

Reply:
removed as per our latest sync